src/handson/Table.js

```diff
@@ -0,0 +1,351 @@
+
+Handsontable.renderers.TextRenderer;
+                        switch (field.type()) {
+                            case "string":
+                                value = field.transform(value);
+                                textRender.call(this, instance, td, row, col, prop, value, cellProperties);
+                                break;
+                            case "number":
+                                Handsontable.renderers.NumericRenderer.call(this, instance, td, row, col, prop, value, 
+
+cellProperties);
+                                break;
+                            case "boolean":
+                                Handsontable.renderers.CheckboxRenderer.call(this, instance, td, row, col, prop, value, 
+
+cellProperties);
+                                break;
+                            case "time":
+                                break;
+                            default:
+                                value = field.transform(value);
+                                textRender.call(this, instance, td, row, col, prop, value, cellProperties);
+                                break;
+                        }
+
+                        if (context._selectionBag.isSelected({ _id: row })) {
+                            td.style.color = "white";
+                            td.style.background = "#f48a00";
+
+                        } else {
+                            td.style.color = null;
+                            td.style.background = null;
+                        }
+                        if (row === context.data().length-1) {
+                               cellProperties.readOnly = false; // make cell read-only if it is first row or the text reads 'readOnly'
+                           if(col === context.columns().indexOf(field.__prop_label)){
+                                cellProperties.formulas = true;
+                                if(value === "sum("+field.__prop_label+")"){
+                                     var temp=0; 
+                                    for(var i=0;i<context.data().length;i++){ 
+                                        var innerArray = context.data()[i]; 
+                                        for(var j=0;j<innerArray.length;j++){ 
+                                                  var cellValue = parseFloat(innerArray[col]);
+                                                 if(typeof cellValue ==="number" && !isNaN(cellValue)){
+                                                    temp += parseFloat(cellValue); 
+                                                    break;
+                                                 }  
+                                        }
+                                    }  
+
+                                    td.textContent = temp;
+                                }
+                           }
+                        }else{
+                                cellProperties.readOnly = true;
```

Comment:
Yes, In last row we can write the formula and it calculates the grand total of the column. sum(column name) gives the total of the column.